coalib/results/TextPosition.py

```diff
@@ -71,11 +70,70 @@ def __ge__(self, other):
         """
         if self.line is None or other.line is None:
             return True
+
         if self.line == other.line:
-            return (True
-                    if self.column is None or
-                    other.column is None
-                    else
-                    self.column >= other.column)
+            if self.column is None or other.column is None:
+                return True
+            else:
+                return self.column >= other.column
         else:
             return self.line > other.line
+
+    def __lt__(self, other):
+        """
+        Test whether ``self`` is behind the other
+        ``TextPosition``.
+
+        If the column in a ``TextPosition`` is ``None``, consider
+        whole line. If the line in a ``TextPosition`` is ``None``,
+        consider whole file.
+
+        :param other: ``TextPosition`` to compare with.
+        :return:      Whether this ``TextPosition`` is behind the other
+                      one.
+        """
+        if self.line is None or other.line is None:
```

Comment:
@userzimmermann so what is your suggestion?

I think to get the correct behaviour here we have to think about when this function will be used and what for. That's where this bug came from in the first place right?